src/NServiceBus.Core/Persistence/Learning/SagaPersister/SagaStorageFile.cs

```diff
@@ -30,7 +30,7 @@ public void Dispose()
             fileStream = null;
         }
 
-        public static Task<SagaStorageFile> Open(Guid sagaId, SagaManifest manifest)
+        public static Task<SagaStorageFile> Open(Guid sagaId, SagaManifest manifest, TimeSpan? maxRetry)
```

Comment:
Why does this retry time need to be configurable? It seems rather unnatural to have to pass this value to `SagaManifestCollection` just to get it down to this class. Could we just pick a reasonable value and always use it?

Reply:
Looking at https://github.com/Particular/NServiceBus/blob/4d991bb0582193b9fef85a86ef0ffd243c5de5f6/src/NServiceBus.PersistenceTests/Sagas/When_concurrent_update_exceed_lock_request_timeout_pessimistic.cs#L11 I figured we needed something but it runs green locally so I removed it. That said something feels fishy here, the test just assumes that 500ms is short enough but I guess that works for now?

Reply:
Tests are green so I guess we're good